### ISSUE TITLE
E143 when installing plugins as a command to nvim

### ISSUE THREAD

**Issue by leesmith** [k-takata/minpac#100]
I've written a bash script to install minpac and my 25 plugins. When running this script, I receive "Error code: 143" for 6 to 8 of the plugins. The number of errors are random...anywhere from 6 to 8. I also just noticed error code 141 in the list below.

> Error while updating "vim-colors-solarized".  Error code: 141
> Error while updating "emmet-vim".  Error code: 143
> Error while updating "vim-eunuch".  Error code: 143
> Error while updating "vim-turbux".  Error code: 143
> Error while updating "vim-ruby".  Error code: 143
> Error while updating "vim-rails".  Error code: 143
> Error while updating "lightline.vim".  Error code: 143
> Error while updating "vim-rake".  Error code: 143
> remote/host: generated rplugin manifest: /Users/lee/.local/share/nvim/rplugin.vim
> Error plugins: 8

> Error while updating "vim-eunuch".  Error code: 143
> Error while updating "lightline.vim".  Error code: 143
> Error while updating "vim-rake".  Error code: 143
> Error while updating "vim-turbux".  Error code: 143
> Error while updating "vim-ruby".  Error code: 143
> Error while updating "vim-colors-solarized".  Error code: 143
> Error while updating "vim-rails".  Error code: 143
> remote/host: generated rplugin manifest: /Users/lee/.local/share/nvim/rplugin.vim
> Error plugins: 7

Here's the script:

```
#!/usr/bin/env bash
#
# Clones minpac vim plugin manager and attempts to install plugins

set -e

if [[ -d "${HOME}/.config/nvim/pack/minpac" ]]; then
  exit 0
else
  [ -d "${HOME}/.config/nvim/pack/minpac/opt" ] || mkdir -p "${HOME}/.config/nvim/pack/minpac/opt"
  cd "${HOME}/.config/nvim/pack/minpac/opt"
  git clone "git://github.com/k-takata/minpac.git" &>/dev/null
  nvim -c 'call minpac#clean()' -c 'call minpac#update()' -c q
  exit 0
fi
```

**Comment by k-takata** [comment]
`minpac#update()` is an asynchronous function. `-c q` quits vim without waiting the end of `minpac#update()`.
See https://github.com/k-takata/minpac#finish-update-hooks how to quit after `minpac#update()`.
Try `nvim -c 'call minpac#clean()' -c 'call minpac#update("", {"do": "quit"})'`.

**Comment by leesmith** [comment]
That's it! Thanks so much.

One nit pick...is there any way to bypass the "Press ENTER or type a command to continue" after the minpac#update?

Thanks again!

**Comment by k-takata** [comment]
You may want to try the 'devel' branch.

**Comment by k-takata** [comment]
I merged the 'devel' branch in to master and released minpac v3.0.0, which doesn't show the message "Press ENTER or type a command to continue".
